src/rpc/mining.cpp

```diff
@@ -492,7 +492,11 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
             std::string lpstr = lpval.get_str();
 
             hashWatchedChain = ParseHashV(lpstr.substr(0, 64), "longpollid");
-            nTransactionsUpdatedLastLP = atoi64(lpstr.substr(64));
+            uint32_t temp;
+            if (!ParseUInt32(lpstr.substr(64), &temp)) {
+                throw JSONRPCError(RPC_INVALID_PARAMETER, "could not parse lpval");
```

Comment:
In commit "refactor: Replace use of atoi64 in rpc/mining.cpp" (b7315cb5114477ede6ebb4e4c2303537dcb647b4)

Slight change of behavior here with stricter parsing. For example no whitespace or trailing characters allowed anymore. But this seems fine and doesn't need additional documentation (though it could be mentioned in the commit message) since there is immediate feedback in case of an error.